frontend/src/components/common/LabelledIcon.tsx

```diff
@@ -32,5 +32,5 @@ const StyledLabelledIcon = styled.div`
 
 const StyledLabel = styled.label`
   font-size: 1.6rem;
-  $font-weight: 700;
+  font-family: 'BCSans=Bold';
```

Comment:
I always wondered whether we had this font available in the assets folder but cannot find it anywhere. Where are we getting it from ?

Reply:
NVM - I found it in package json

Reply:
there is a typo here however, will fix.